contrib/src/main/java/com/datatorrent/contrib/hbase/AbstractHBaseWindowAppendOutputOperator.java

```diff
@@ -63,50 +55,35 @@
  *            The tuple type
  * @since 1.0.2
  */
-public abstract class AbstractHBaseWindowAppendOutputOperator<T> extends AbstractAggregateTransactionableStoreOutputOperator<T, HBaseWindowStore> {
+public abstract class AbstractHBaseWindowAppendOutputOperator<T> extends AbstractHBaseWindowOutputOperator<T> {
   private static final transient Logger logger = LoggerFactory.getLogger(AbstractHBaseWindowAppendOutputOperator.class);
-  private List<T> tuples;
   private transient ProcessingMode mode;
+  
+  @Deprecated
   public ProcessingMode getMode()
   {
     return mode;
   }
 
+  @Deprecated
   public void setMode(ProcessingMode mode)
   {
     this.mode = mode;
   }
 
   public AbstractHBaseWindowAppendOutputOperator() {
     store = new HBaseWindowStore();
-    tuples = new ArrayList<T>();
   }
 
   @Override
-  public void storeAggregate() {
```

Comment:
By getting rid of this method/logic we are no longer supporting batch/window mode. Why can't we keep this method and do the following?
```
   Iterator<T> it = tuples.iterator();		
   while (it.hasNext()) {		
       T t = it.next();
       HTable table = store.getTable(getTableName(t));
       try {		
         Append append = operationAppend(t);		
         table.append(append);		
       } catch (IOException e) {		
         logger.error("Could not output tuple", e);		
         DTThrowable.rethrow(e);		
       }				
     }
```
Coupled with my comments on Adapter/Driver this may become even easier.